src/thunderbird_accounts/mail/views.py

```diff
@@ -53,6 +53,12 @@ def sign_up_submit(request: HttpRequest):
 
     email_address = f'{request.POST['email_address']}@{request.POST['email_domain']}'
```

Comment:
Hmm I guess we may want to make a ticket to further sanitize this email address input. (No @'s, no spaces, etc..)

Reply:
Good idea, I created issue #85.

Thanks for the review!